Documentation/ApiOverview/Database/QueryBuilder/Index.rst

```diff
@@ -690,6 +690,10 @@ Further remarks:
     involved tables for equality and implement a fallback logic in PHP if they
     are different.
 
+*   Doctrine DBAL does not support the use of join methods in combination with
```

Comment:
shall we make this a tip admonition?

Reply:
I think it is not more important than the other remarks. We had this discussion a few days ago in Slask, and so I think this should be mentioned.